data.sql

```diff
@@ -16,4 +16,50 @@ VALUES
   (8, 'Angemon', '2005-06-12', 1, true, -45),
   (9, 'Boarmon', '2005-06-07', 7, true, 20.4),
   (10, 'Blossom', '1998-10-13', 3, true, 17),
-  (11, 'Ditto', '2022-05-14', 4, true, 22);
\ No newline at end of file
+  (11, 'Ditto', '2022-05-14', 4, true, 22);
+
+INSERT INTO owners(full_name, age)
+VALUES
+('Sam Smith', 34 ),
+('Jennifer Orwell', 19 ),
+('Bob ', 45 ),
+('Melody Pond', 14 ),
+('Melody Pond', 14 ),
```

Comment:
- [ ] The **Melody Pond** data is duplicated Kindly remove one.
- [ ] Nice work inserting the data into the `owers` table :clap: However, According to the project requirement we should add **Dean Winchester 14 years old**, but this owner is not present in your `data.sql` file. Kindly make sure you add the query that you use to insert each owner to the `data.sql` file to follow the project requirement.